contrib/spec/podman.spec.in

```diff
@@ -435,7 +435,7 @@ BUILDTAGS=$BUILDTAGS make binaries
 %install
 install -dp %{buildroot}%{_unitdir}
 install -dp %{buildroot}%{_usr}/lib/systemd/user
-PODMAN_VERSION=%{version} %{__make} PREFIX=%{buildroot}%{_prefix} ETCDIR=%{buildroot}%{_sysconfdir} \
+PODMAN_VERSION=%{version} %{__make} DESTDIR=%{buildroot} PREFIX=/usr ETCDIR=%{_sysconfdir} \
```

Comment:
@lsm5 PTAVCL (Very Close Look) at this change. I'm highly confident that it is the right thing to do, and that this fixes something that has been broken from Day One... but when I get "confident" is exactly when I need to ask for extra sets of eyes. I don't know who/what else uses this specfile, or what cascading effects this might have.TIA.

Reply:
Why is PREFIX=/usr ?  Currently make install installs in /usr/local.  The exact problem is, that make install installs in /usr/local/bin/podman, but the service files looked at /usr/bin/podman wihtout /local.